README.md

```diff
@@ -63,10 +63,7 @@ To stop the server run:
 
 To remove all containers run:
 
-    $ docker-compose rm
-
-To remove the database you have to remove the content of the `db-data`
-directory.
+    $ docker-compose down --volumes
```

Comment:
I would like to keep two commands here. One to remove the container without the volume and one to remove both (with an explicit hint that this will remove the whole database and all data would be lost ...